File: loop/1066.py
#1066 PARES, ÍMPARES, POSITIVOS E NEGATIVOS
pares = 0
impares = 0
positivos = 0
negativos = 0

for _ in range(5):
    i = float(input())
    if i%2 == 0:
        pares += 1
    elif i%2 != 0:
        impares += 1

    if i > 0:
        positivos += 1
    elif i < 0:
        negativos += 1

print(f"{pares} valor(es) par(es)")
print(f"{impares} valor(es) impar(es)")
print(f"{positivos} valor(es) positivo(s)")
print(f"{negativos} valor(es) negativo(s)")

File: javascript/1066.js
/*Pares,Ímpares, Positivos e Negativos*/
let input = require("fs").readFileSync("/dev/stdin", "utf8");

let lines = input.split("\n");

valores = 0;
numPar = 0;
numImpar = 0;
numPositivo = 0;
numNegativo = 0;

for(let i = 0; i < 5; i++){
  valores = parseInt(lines.shift());
   if(valores % 2 === 0){
    numPar++;
  }else{
    numImpar++;
  }
    if(valores > 0){
    numPositivo++;
  }else if(valores < 0){
    numNegativo++;
  } 
}
console.log(numPar + " valor(es) par(es)");
console.log(numImpar + " valor(es) impar(es)");
console.log(numPositivo + " valor(es) positivo(s)");
console.log(numNegativo + " valor(es) negativo(s)");